client/src/pages/Restaurant/RestaurantMap/Map.js

```diff
@@ -0,0 +1,44 @@
+/*global google*/
+import React from "react";
+import { GoogleMap } from "@react-google-maps/api";
+
+const containerStyle = {
+  width: "100%",
+  height: "80vh",
```

Comment:
Too big. Make this smaller.
Ideally it shouldn't take up more than half the screen (even on mobile). Try 40vh.

This is component supposed to serve as a decoration/preview instead of a functional map